### PR TITLE
New package: boilr-0.3.0

### PR DESCRIPTION
**Pull request by mahnve**

### CODE REVIEW

File: srcpkgs/boilr/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'boilr'
+pkgname=boilr
+version="0.3.0"
```
Comment (maxice8) [inline srcpkgs/boilr/template]: unquote

File: srcpkgs/boilr/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+revision=1
+build_style=go
+only_for_archs="x86_64"
```
Comment (maxice8) [inline srcpkgs/boilr/template]: why ?

File: srcpkgs/boilr/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+hostmakedepends="git"
+short_desc="Generates files or directories from template repositories"
+maintainer="Marcus Ahnve <m@hnve.org>"
+license="Apache 2.0"
+go_import_path="github.com/tmtrs/${pkgname}"
```
Comment (maxice8) [inline srcpkgs/boilr/template]: put directly below build_style=

File: srcpkgs/boilr/template (hunk 2 above)
Comment (mahnve) [reply srcpkgs/boilr/template]: Sorry, my mistake. Had newbie problems compiling and tried that as a workaround. Forgot to remove.

**Comment by mahnve** [comment]
Turns out that there's a bug in the package that's not released, which prevents building. Closing.

**Pull request closed without merging**